app/routes/index.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const { service } = Ember.inject;
+
+export default Ember.Route.extend({
+  mainMap: service(),
+
+  actions: {
+    didTransition() {
+      this.get('mainMap').set('selected', null);
+      this.get('mainMap').set('shouldFitBounds', false);
+    },
```

Comment:
I'll take care of this but if you're setting on a property you don't need to get it, you can just write:
`this.set('mainMap.selected',null);`

Alternatively, if there are multiple properties, you can just say:
```
      this.get('mainMap')
        .setProperties({
          selected: null,
          shouldFitBounds: false,
        });
```

Reply:
good to know!